assets/sass/components/adsense/_index.scss

```diff
@@ -15,3 +15,4 @@
  */
 
 @import "googlesitekit-adsense-ad-blocker-recovery";
+@import "googlesitekit-adsense-ad-blocking-recovery-notification";
```

Comment:
@nfmohit, could you please rename the `_googlesitekit-adsense-ad-blocker-recovery.scss` file to be `_googlesitekit-adsense-ad-blocking-recovery.scss` and add new styles to it instead of creating a new file?

Reply:
I've made the changes you've suggested here. There are other places where `adBlockerRecovery` and `ad-blocker-recovery` is used instead of `blocking` recovery, which I've kept as is. I'm assuming this will be fixed in a different issues (I remember an ABD IB mentioning this change). 

Cheers.

Reply:
Thanks, @kuasha420. Yes, we have another ticket that will rename that.